mitodl/odl-video-service
# Add dialog for creating a new collection

#### What are the relevant tickets?
Fixes #227

#### What's this PR do?
Refactors the collection edit dialog to also be used in creating a new collection, and removes the collection form

#### How should this be manually tested?
 - Create a new collection using the button in the drawer. The collection should be created successfully and it should go on the beginning of the collection list. Make sure to add items to the moira list.
 - Create a new collection using the 'Create Collection' button at `/collections/`. You should see the same user experience.
 - Edit one of the collections and verify that this produces the expected change.
